main.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"os"
+	"text/template"
+
+	"muzzammil.xyz/jsonc"
+)
+
+// Resource definityion for the package
+type Resource struct {
+	Name   string  `json:"name"`
+	Length *Length `json:"length,omitempty"`
+	Regex  *string `json:"regex,omitempty"`
+	Scope  *string `json:"scope,omitempty"`
+	Slug   *string `json:"slug,omitempty"`
+	Dashes bool    `json:"dashes"`
+}
+
+// Length allowed for that resorce
+type Length struct {
+	Min int `json:"min"`
+	Max int `json:"max"`
+}
+
+func main() {
+	files, err := ioutil.ReadDir("templates")
+	if err != nil {
+		log.Fatal(err)
+	}
+	var fileNames = make([]string, 0)
```

Comment:
No biggy here, but might as well use:
```suggestion
	var fileNames = make([]string, len(files))
```

Reply:
This would cause an error if I was still using append so I changed to also do not use append.